package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "muskie",
     "description": "Manta public-facing WebAPI",
-    "version": "1.0.2",
+    "version": "2.0.0",
```

Comment:
do you want to 2.0.0 package.json version change in this one? That'll break things without the lib/configure.js `version()` change, right?

Reply:
yep. fine.